packages/flutter/test/material/menu_anchor_test.dart

```diff
@@ -4508,6 +4508,47 @@ void main() {
       expect(state.target, isNull);
     }, skip: kIsWeb // [intended] ForceGC does not work in web and in release mode. See https://api.flutter.dev/flutter/package-leak_tracker_leak_tracker/forceGC.html
   );
+
+  // Regression test for https://github.com/flutter/flutter/issues/154842
+  testWidgets('SubmenuButton mouse onHover is not triggered on Android', (WidgetTester tester) async {
```

Comment:
This test doesn't fail before the fix as the `MouseRegion` callbacks aren't triggered in the test. Only when running the sample on Android emulator or device, these MouseRegion` callbacks are unexpectedly triggered and causing the bug.

Reply:
cc: @jason-simmons